### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cozyplanes**
added korean translator contributor, and some new FAQs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,7 +154,21 @@ Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed inf
   <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it,
   in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b>
   cases your GitHub account Oauth token will do the trick.</p>
+</details>
+
+<details>
```
Comment (yakov116) [inline README.md]: editing an issue is a public...

Comment (cozyplanes) [reply README.md]: so sorry, but i don't understand what you are talking about....

Comment (k0shk0sh) [reply README.md]: Test

Comment (cozyplanes) [reply README.md]: @yakov116 reply, please? I don't understand what is going on  here

Comment (yakov116) [reply README.md]: @cozyplanes I will reply in about an hour

File: README.md [hunk 2]
```diff
@@ -139,21 +141,33 @@ Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed inf
...
-  <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it,
-  in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b>
-  cases your GitHub account Oauth token will do the trick.</p>
+  <summary>I login with Enterprise account but can't interact with anything other than my Enterprise GitHub.</summary>
+  <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it, in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b> cases your GitHub account Oauth token will do the trick.</p>
+</details>
+
+<details>
+  <summary>Why am I having problems editing Issues/PRs?</summary>
+  <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
```
Comment (yakov116) [inline README.md]: Over here. Change the text to "If you are unable to edit an **issue** in a public...."

Comment (cozyplanes) [reply README.md]: Ok!

**Comment by cozyplanes** [comment]
merge time! i guess

**Comment by yakov116** [comment]
4 commits LOL
This is why i never work on github directly.

I'm going to quash and merge

**Comment by cozyplanes** [comment]
@yakov116 great job sir!

**Pull request merged**